runtime/src/main/scala/akka/grpc/internal/NettyClientUtils.scala

```diff
@@ -57,6 +58,10 @@ object NettyClientUtils {
               .getOrElse(builder.negotiationType(NegotiationType.PLAINTEXT))
           }
 
+          builder = settings.grpcLoadBalancingType
+            .map(builder.defaultLoadBalancingPolicy(_))
+            .map(_ => builder.nameResolverFactory(new DnsNameResolverProvider()))
```

Comment:
This indeed works because the build methods are side-effecting, but wouldn't it be slightly neater to:

```suggestion
            .map(_.nameResolverFactory(new DnsNameResolverProvider()))
```

?

Reply:
Sure!